### ISSUE TITLE
Focus the first text input on the page - lack of scroll to active input field

### ISSUE THREAD

**Issue by szuri** [philc/vimium#4046]
If you type "gi" to focus the first text input on the page, in case scenario when the input field is not visible your screen at the moment of typing keystroke would be nice to implement automatic scroll to activated input filed position of a page.

In version 1.67  after type "gi" you must set scroll position manually (if input filed is outside the currently displayed user's view) or typing text into input field blindfold - you can't see what you typed.

**Comment by YunfangHou** [comment]
Yeah, I have the same feature request. For reference [Vimari](https://github.com/televator-apps/vimari) supports this feature.
